pom.xml

```diff
@@ -260,6 +260,22 @@
                     <useSystemClassLoader>false</useSystemClassLoader>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>properties-maven-plugin</artifactId>
+                <version>1.0.0</version>
+                <executions>
+                    <execution>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>write-project-properties</goal>
+                        </goals>
+                        <configuration>
+                            <outputFile>${project.build.directory}/classes/properties-from-pom.properties</outputFile>
```

Comment:
How is this used?

Reply:
In order to know in code, what is the api version, I copied over the pom properties including the property `api-spec-yaml-file` using this plugin, and then read the property in code.